pkg/workloads/workload_integration_test.go

```diff
@@ -0,0 +1,77 @@
+// +build integration
+
+package workloads
+
+import (
+	"os"
+	"testing"
+
+	"github.com/newrelic/newrelic-client-go/pkg/config"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	testWorkloadName  = "testWorkload"
+	testWorkloadQuery = "(name like 'tf_test' or id = 'tf_test' or domainId = 'tf_test')"
+	testAccountID     = 2508259
+	testCreateInput   = CreateInput{
+		Name: &testWorkloadName,
+		ScopeAccountsInput: ScopeAccountsInput{
+			AccountIDs: []*int{&testAccountID},
+		},
+		EntitySearchQueries: []*EntitySearchQueryInput{
+			{
+				Name:  "testQuery",
+				Query: &testWorkloadQuery,
+			},
+		},
+	}
+)
+
+func TestIntegrationGetWorkload(t *testing.T) {
+	t.Parallel()
+
+	client := newIntegrationTestClient(t)
+
+	actual, err := client.GetWorkload(testAccountID, 791)
+
+	require.NoError(t, err)
+	require.NotNil(t, actual)
+}
+
+func TestIntegrationListWorkloads(t *testing.T) {
+	t.Parallel()
+
+	client := newIntegrationTestClient(t)
+
+	actual, err := client.ListWorkloads(2508259)
+
+	require.NoError(t, err)
+	require.Greater(t, len(actual), 0)
+}
+
+func TestIntegrationCreateWorkload(t *testing.T) {
+	t.Parallel()
+
+	client := newIntegrationTestClient(t)
+
+	actual, err := client.CreateWorkload(testAccountID, &testCreateInput)
+
+	require.NoError(t, err)
+	require.NotNil(t, actual)
+}
+
+// nolint
+func newIntegrationTestClient(t *testing.T) Workloads {
```

Comment:
How often do we use this? I know I've seen it before in this (maybe?) repo.  Might be worth moving to `internal/testing` package or what not (ala `newrelic-cli/internal/testcobra` package)

Reply:
It's specific to the `workloads` package, though.  I like the idea of keeping all that code together.

Reply:
Dah, right, I completely missed the return value...